packages/adapter-netlify/README.md

```diff
@@ -36,6 +36,15 @@ Then, make sure you have a [netlify.toml](https://docs.netlify.com/configure-bui
 
 It's recommended that you add the `build` and `functions` folders (or whichever other folders you specify) to your `.gitignore`.
 
-## Extra Resources
+## Troubleshooting
 
-Guide for SvelteKit + Netlify Forms users: https://dev.to/swyx/how-to-use-sveltekit-with-netlify-forms-5gmj
+### Custom Netlify redirect rules
+
+During compilation a required "catch all" redirect rule is automatically appended to your `_redirects` file. (If it doesn't exist yet, it will be created.) That means:
+
+- `[[redirects]]` in `netlify.toml` will never match as `_redirects` has a [higher priority](https://docs.netlify.com/routing/redirects/#rule-processing-order). So instead put your rules in the [`_redirects` file](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file).
+- `_redirects` cannot have any custom "catch all" rules such as `/* /foobar/:splat`. Else the one appended by the Netlify Adapter will never be applied because only [the first matching rule](https://docs.netlify.com/routing/redirects/#rule-processing-order) is processed.
+
+### Guide for SvelteKit + Netlify Forms users
```

Comment:
let's leave this out of this PR and we can handle separately in https://github.com/sveltejs/kit/pull/1481